airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -82,32 +83,113 @@ class BigQueryHook(GoogleBaseHook, DbApiHook):
 
     This hook uses the Google Cloud connection.
 
-    :param gcp_conn_id: The Airflow connection used for GCP credentials.
     :param use_legacy_sql: This specifies whether to use legacy SQL dialect.
     :param location: The location of the BigQuery resource.
     :param priority: Specifies a priority for the query.
         Possible values include INTERACTIVE and BATCH.
         The default value is INTERACTIVE.
     :param api_resource_configs: This contains params configuration applied for
         Google BigQuery jobs.
-    :param impersonation_chain: This is the optional service account to
```

Comment:
Both `gcp_conn_id` and `impersonation_chain` exist in the parent

Reply:
Unfortunetly docstring doesn't work by that way, so it would be missed into the IDE suggestion as well into the API reference

Reply:
I restored these params, but I'd be happy if you could elaborate. In general, when writing a docstring for a child class, should we also document attributes from its parent? On the one hand, it sounds comfortable when referring to the API; on the other hand, it quite violates DRY.